### ISSUE TITLE
ErrorBoundary prop-type checking logs warning for missing intl

### ISSUE THREAD

**Issue by StephenEsser** [cerner/terra-application#48]
# Bug Report

## Description

ErrorBoundary prop-type checking logs warning for missing intl.

```
Warning: Failed prop type: The prop `intl.isRequired` is marked as required in `ApplicationErrorBoundary`, but its value is `undefined`.
    in ApplicationErrorBoundary (created by InjectIntl(ApplicationErrorBoundary))
    in InjectIntl(ApplicationErrorBoundary) (created by ApplicationBase)
    in IntlProvider (created by I18nProvider)
    in I18nProvider (created by Base)
    in Base (created by ApplicationBase)
    in ThemeContextProvider (created by ApplicationBase)
    in ThemeProvider (created by ApplicationBase)
    in div (created by ApplicationBase)
    in ApplicationBase (created by Context.Consumer)
    in AppSettingsProvider (created by Site)
    in Site (created by TerraDevSite)
    in TerraDevSite
```
## Steps to Reproduce

1. Clone terra-application
2. Start site `npm start` and navigate to localhost:8080 or the raw demo page (http://localhost:8080/raw/application/terra-application/demo)
3. Observe the warning in the console

## Additional Context / Screenshots

~This may have something to do with terra-base lazy loading intl prior to the ErrorBoundary prop-type validation has completed.~

Carrying forward Tyler's recommendation from below: 

> https://github.com/cerner/terra-application/blob/master/src/application-error-boundary/ApplicationErrorBoundary.jsx#L18

> I think this should just be intl: intlShape, without the additional shape around it.

## Expected Behavior

The warning should not be logged to the console.

## Possible Solution

-- 

## Environment

* Component Name and Version:
* Browser Name and Version:
* Node/npm Version: [e.g. Node 8/npm 5]
* Webpack Version:
* Operating System and version (desktop or mobile):

**Comment by tbiethman** [comment]
https://github.com/cerner/terra-application/blob/master/src/application-error-boundary/ApplicationErrorBoundary.jsx#L18

I think this should just be `intl: intlShape,` without the additional shape around it.
